index.js

```diff
@@ -18,22 +19,29 @@ const SUBMISSION = [
 
 ]
 
+app.use(express.json())
+
 app.post('/signup', function(req, res) {
   // Add logic to decode body
   // body should have email and password
-
-
   //Store email and password (as is for now) in the USERS array above (only if the user with the given email doesnt exist)
-
-
   // return back 200 status code to the client
-  res.send('Hello World!')
+  const {email, password} = req.body;
+  const user = USERS.find(u => u.email === email && u.password === password);
```

Comment:
According to my understanding, only email check is sufficient because users can have the same password.